test/composite.test.js

```diff
@@ -11,5 +11,5 @@ test('anonymous function should be named composite.test', t => {
     next()
   })
 
-  t.is(fn[Symbol.for('fastify.display-name')], 'composite.test')
```

Comment:
Can you keep `fastify.display-name` as well? Otherwise it will be a breaking change, and I would not bump the major for this.